tests/test_s1_geocode_slc.py

```diff
@@ -117,4 +117,4 @@ def test_geocode_slc_validate(geocode_slc_params):
 
     # check for bright spots in sliced array
     corner_reflector_threshold = 3e3
-    assert np.any(arr > corner_reflector_threshold)
+    assert np.any(np.abs(arr) > corner_reflector_threshold)
```

Comment:
Here is the suggested change in the test code. 
It looks like `complex64` is being compared to `float`. In this case, it seems numpy only takes the real part of the complex number and to the comparison. I think it makes more sense to calculate the magnitude of the complex image. @LiangJYu What do you think?


> np.array([3001 + 9000 * 1.0j]) > 3e3
array([ True])
> np.array([2001 + 9000 * 1.0j]) > 3e3
array([False])
> np.array([2001 + 1000 * 1.0j]) > 3e3
array([False])
> np.array([3001 + 1000 * 1.0j]) > 3e3
array([ True])

Reply:
Good find! Please implement your proposed fix.